app/docker-compose.prod.yml

```diff
@@ -60,7 +61,18 @@ services:
     networks:
       - media_nginx
       - envoy_nginx
+  prometheus:
+    build:
+      context: prometheus
```

Comment:
Needs to be an `image: ` in prod, no building there

Reply:
good find, fixing this